game/resource/English/modifier/items/modifier_greater_arcane_boots.txt

```diff
@@ -0,0 +1,5 @@
+//=============================================================================
+// Greater Arcanes Ticking GPM Value
+//=============================================================================
+"DOTA_Tooltip_modifier_greater_arcane_boots"                   "Inflation"
+"DOTA_Tooltip_modifier_greater_arcane_boots_Description"       "Passive gold income reduced by %dMODIFIER_PROPERTY"
```

Comment:
I know you probably left this in as something to look into later, but anyway... Should be %dMODIFIER_PROPERTY_TOOLTIP%. You can then set the value by making an `OnTooltip` function that returns the value you want to display. Though I'm unsure if it actually updates dynamically.

Reply:
thanks, i actually had no clue how to do this so this is helpful